.gitignore

```diff
@@ -13,6 +13,9 @@ Makefile.in
 *.lo
 *.a
 *.la
+.cproject
+.project
+.settings/language.settings.xml
 
```

Comment:
@kjankov Why is this here? I mean I'm OK with this, but I wonder why is this needed? Do you use some special cool editor? Maybe you use something better than I'm using :-) Thanks for reply.